public/js/app/pages/AboutCourseMemo.jsx

```diff
@@ -135,19 +120,16 @@ function isCurrentMemoIsUnqiue(memoList, round, memoToCheck) {
         return false
       }
     }
-  }
-  if (memo) {
     refMemos.push(memo)
   }
   memoToCheck.current = refMemos
   return true
 }
 
 function extendMemo(memo, round) {
-  if (memo.isPdf !== true || (memo.isPdf === true && memo.applicationCodes.length == 1))
-    memo.shortName = round.shortName
+  memo.shortName = round.shortName
   memo.firstTuitionDate = round.firstTuitionDate
-
+  memo.term = round.term
```

Comment:
Why is memo.term is added here? memo has already own `semester` field which is the same as `term` field, and memo's semester has to stay as the source of truth for memos. So this line should be removed

Reply:
Term is removed here.